SConstruct

```diff
@@ -131,6 +131,7 @@ opts.Add(BoolVariable("xaudio2", "Enable the XAudio2 audio driver", False))
 opts.Add(BoolVariable("vulkan", "Enable the vulkan video driver", True))
 opts.Add("custom_modules", "A list of comma-separated directory paths containing custom modules to build.", "")
 opts.Add(BoolVariable("custom_modules_recursive", "Detect custom modules recursively for each specified path.", True))
+opts.Add(BoolVariable("use_volk", "Use the volk library", True))
```

Comment:
```suggestion
opts.Add(BoolVariable("use_volk", "Use the volk library to load Vulkan drivers", True))
```

Reply:
Loading a loading is weird, but that's what it does :P

Reply:
Done